<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgLastCurrent</class>
 <widget class="QWidget" name="wdgLastCurrent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>524</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Last operation investments report</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Percentage gains got in last operation investment</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spin">
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="minimum">
          <number>-100</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>-33</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="mqtwObjects" name="mqtwInvestments" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Select the way you want to see results</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbSameProduct">
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>Show separated investments</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Show merging current investment operations</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Show merging all investment operations</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="wdgIBM" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionInvestmentReport">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/bundle.png</normaloff>:/xulpymoney/bundle.png</iconset>
   </property>
   <property name="text">
    <string>Investment report</string>
   </property>
   <property name="toolTip">
    <string>Investment report</string>
   </property>
  </action>
  <action name="actionSortTPCVenta">
   <property name="icon">
    <iconset>
     <normaloff>:/xulpmoney/document-preview.png</normaloff>:/xulpmoney/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>% Selling point</string>
   </property>
   <property name="toolTip">
    <string>% Selling point</string>
   </property>
  </action>
  <action name="actionSortTPC">
   <property name="icon">
    <iconset>
     <normaloff>:/xulpmoney/document-preview.png</normaloff>:/xulpmoney/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>% Invested</string>
   </property>
   <property name="toolTip">
    <string>% Invested</string>
   </property>
  </action>
  <action name="actionSortName">
   <property name="icon">
    <iconset>
     <normaloff>:/xulpmoney/document-preview.png</normaloff>:/xulpmoney/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="toolTip">
    <string>Name</string>
   </property>
  </action>
  <action name="actionSortHour">
   <property name="icon">
    <iconset>
     <normaloff>:/xulpmoney/document-preview.png</normaloff>:/xulpmoney/document-preview.png</iconset>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
   <property name="toolTip">
    <string>Time</string>
   </property>
  </action>
  <action name="actionProduct">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/books.png</normaloff>:/xulpymoney/books.png</iconset>
   </property>
   <property name="text">
    <string>Product report</string>
   </property>
   <property name="toolTip">
    <string>Product report</string>
   </property>
  </action>
  <action name="actionSortTPCLast">
   <property name="text">
    <string>% Last operation</string>
   </property>
   <property name="toolTip">
    <string>% Last operation</string>
   </property>
  </action>
  <action name="actionCalculate">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/tools-wizard.png</normaloff>:/xulpymoney/tools-wizard.png</iconset>
   </property>
   <property name="text">
    <string>Calculate reinvest price</string>
   </property>
   <property name="toolTip">
    <string>Calculate reinvest price</string>
   </property>
  </action>
  <action name="actionReinvest">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/coins.png</normaloff>:/xulpymoney/coins.png</iconset>
   </property>
   <property name="text">
    <string>Reinvest</string>
   </property>
   <property name="toolTip">
    <string>Reinvest</string>
   </property>
  </action>
  <action name="actionReinvestCurrent">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/coins.png</normaloff>:/xulpymoney/coins.png</iconset>
   </property>
   <property name="text">
    <string>Simulate reinvestment at product current price</string>
   </property>
   <property name="toolTip">
    <string>Simulate reinvestment at product current price</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqtwObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
